cmd/gaiacli/README.md

```diff
@@ -0,0 +1,195 @@
+
+```shell
+gaiacli \
+  --home ibc0/n0/gaiacli \
+  tx ibc connection handshake \
+  conn0 c0 $(gaiacli --home ibc1/n0/gaiacli q ibc client path) \
+  conn1 c1 $(gaiacli --home ibc0/n0/gaiacli q ibc client path) \
+  --chain-id2 ibc1 \
+  --from1 n0 --from2 n1 \
+  --node1 tcp://localhost:26657 \
+  --node2 tcp://localhost:26557
+```
+
+Once the connection is established you should be able to query it:
+
+```bash
+gaiacli --home ibc0/n0/gaiacli q ibc connection connection conn0 --indent --trust-node
+gaiacli --home ibc1/n0/gaiacli q ibc connection connection conn1 --indent --trust-node
+```
+
+### Channel
+
+Now that the `connection` has been created, its time to establish a `channel` for the `ibc-mock` application protocol. This will allow sending of data between `ibc0` and `ibc1`. To create the `channel`, run the following command:
+
+> NOTE: This command broadcasts a total of 7 transactions between the two chains from 2 different wallets. At the start of the command you will be prompted for passwords for the two different keys. The command may then take some time. Please wait for the command to return!
+
+```bash
+gaiacli \
+  --home ibc0/n0/gaiacli \
+  tx ibc channel handshake \
+  ibcmocksend chan0 conn0 \
+  ibcmockrecv chan1 conn1 \
+  --node1 tcp://localhost:26657 \
+  --node2 tcp://localhost:26557 \
+  --chain-id2 ibc1 \
+  --from1 n0 --from2 n1
+```
+
+You can query the `channel` after establishment by running the following command:
+
+```bash
+gaiacli --home ibc0/n0/gaiacli query ibc channel channel ibcmocksend chan0 --indent --trust-node
+gaiacli --home ibc1/n0/gaiacli query ibc channel channel ibcmockrecv chan1 --indent --trust-node
+```
+
+## Send Packet
+
+To send a packet using the `ibc-mock` application protocol, you need to know the channel you plan to send on, as well as the sequence number on the channel. To get the sequence you use the following commands:
+
+```bash
+# Returns the last sequence number
+gaiacli --home ibc0/n0/gaiacli q ibcmocksend sequence chan0
+
+# Returns the next expected sequence number, for use in scripting
+gaiacli --home ibc0/n0/gaiacli q ibcmocksend next chan0
+```
+
+Now you are ready to send an `ibc-mock` packet down the channel (`chan0`) from chain `ibc0` to chain `ibc1`! To do so run the following command:
+
+```bash
+gaiacli --home ibc0/n0/gaiacli tx ibcmocksend sequence chan0 $(gaiacli --home ibc0/n0/gaiacli q ibcmocksend next chan0) --from n0 -o text
+```
+
+### Receive Packet
+
+Once packets are sent, reciept must be confirmed on the destination chain. To receive the packets you just sent, run the following command:
+
+```bash
+gaiacli \
+  --home ibc0/n0/gaiacli \
+  tx ibc channel flush ibcmocksend chan0 \
+  --node1 tcp://localhost:26657 \
+  --node2 tcp://localhost:26557 \
+  --chain-id2 ibc1 \
+  --from1 n0 --from2 n1 -y -o text
```

Comment:
The `-y` option doesn't seem to exist for this build and running this command results in this error:
`ERROR: unknown shorthand flag: 'y' in -y`

Ran `gaiacli --home ibc0/n0/gaiacli tx ibc channel flush ibcmocksend --help` and didn't see the option/flag listed either.